docs/elements/hx-pill/index.html

```diff
@@ -51,18 +51,18 @@ <h2 id="methods">Methods</h2>
 
 {% block attributes %}
   <dl>
-    <dt>dismissable {Boolean [false]} <i>(optional)</i></dt>
+    <dt>persist {Boolean}</dt>
     <dd>
-      When present, pill will be dismissable.
+      When present, pill will be not dismissable.
```

Comment:
suggest changing this to: `When present, pill will not be dismissable.`

Reply:
good catch, updated!!